src/core/validators/validators.ts

```diff
@@ -144,9 +158,9 @@ const Url: ValidatorFn = (value) => {
   return Promise.resolve(null);
 };
 
-const isEmptyValue = (value: Value) => {
-  return  _.isEmpty(value) || (typeof value==='string' && value?.trim?.() === '');
-}
+const isEmptyValue = (value: Value) =>
+  (_.isEmpty(value) && !_.isDate(value)) ||
+  (_.isString(value) && value?.trim?.() === '');
```

Comment:
if trim exists then it's a string... so second statement includes first right?

Reply:
yes